SequenceOfNumbersModified.java

```diff
@@ -0,0 +1,42 @@
+import java.util.Scanner;
+
+public class SequenceOfNumbersModified {
+  /** 
+   * The function is used for determination type of sequence: non-decreasing or not non-decreasing
+   *
+   * @param newArgs - array of strings which contain testing sequence. One elemet of array - one number
+   */ 
+  public void сheckSequence(String[] newArgs) {
+    if (newArgs.length == 1) {
+      System.out.println("You entered only one number. It is not a sequence");

```

Comment:
single responsibility principle violation - the method must check and return result but not check and output result.